src/constants.js

```diff
@@ -94,4 +94,10 @@ module.exports = {
     CREATE_EVENT: 3500000,
     DORACLE_CREATE: 1500000,
   },
+
+  DBVoteType: {
```

Comment:
just `VoteType` is better since we are using the same type for both backend/UI. better to keep it consistent across all stacks.